Doc/library/pdb.rst

```diff
@@ -75,25 +75,36 @@ The debugger's prompt is ``(Pdb)``, which is the indicator that you are in debug
    arguments of the ``p`` command.
 
 
+.. program:: pdb
+
 You can also invoke :mod:`pdb` from the command line to debug other scripts.  For
-example::
+example:
+
+.. code-block:: bash
 
-   python -m pdb myscript.py
+   python -m pdb [-c command] (-m module | pyfile) [args ...]
```

Comment:
I don't think it makes a huge difference but it seems like, for cProfile and ensurepip, the example line was not in a `.. code-block::` and it kept that way. For Idle, it was in a `.. code-block::` and it also kept that way. Maybe we should just keep this line the old way without the `.. code-block::`? I'm not the docs expert, just trying to follow the patterns.

Reply:
I'm not docs expert either, but it seems like `.. code-block::` is the new and preferred way

Reply:
Is there any referece to this statement? I can believe it but it would be nice to have a confirmation.

Reply:
to be honest I used it because sobolevn also used it in his [PR](https://github.com/python/cpython/pull/130497/files)

as far as I understood the only difference is that the directive `.. code-block::` adds syntax highlighting and sphinx understands the language

Reply:
If you want, let's back to previous syntax. It doesn't matter to me

Reply:
In this specific case, there won't be any difference but semantically it can be helpful (for instance if we customize the Pygments or if pygments itself changes how to render console-like lines).

Using :: means using a plain code-block without syntax highlighting IIRC.

Reply:
I put it back the way it was before. I don't want to pay too much attention to it.